native/spark-expr/src/lib.rs

```diff
@@ -24,7 +24,7 @@ mod temporal;
 pub mod timezone;
 pub mod utils;
 
-pub use cast::Cast;
+pub use cast::{Cast, spark_cast_array};
```

Comment:
the export is not needed by comet, but it allows others (eg. us) to use spark_cast_array directly